rootfs/lua/avaya_validate_path.lua

```diff
@@ -0,0 +1,43 @@
+
+local _M = {}
+
+-- alternatively: local lrucache = require "resty.lrucache.pureffi"
+local lrucache = require "resty.lrucache"
+local cache_ttl_seconds = 300 -- five minutes
+
+-- we need to initialize the cache on the lua module level so that
+-- it can be shared by all the requests served by each nginx worker process:
+local c = lrucache.new(10000)  -- allow up to 10k items in the cache
```

Comment:
We may want more than 10k if it doesn't use too much memory. There are currently over 10k accounts in our system and this is caching based on each url. Perhaps we should cache based on the account sid...

Reply:
Hmmm, I see your point. I thought of using the URL path as a key to keep the cache search flat(having a key per account means we need to cache a structure like `{sid: [path1, path2, path3]`.
Maybe we could just increase the cache to 100k entries?
The memory consumed would still be the same, the difference is we would store a single key per path instead of nesting it into an array.